part3.md

```diff
@@ -0,0 +1,80 @@
+## Part 3 - Hello Kotlin
+
+ * Make sure that you have Kotlin JS installed (in `functions` sub-folder)
```

Comment:
does it have to be installed in the functions subfolder? how do i ensure that? or do i have to run the command in the subfolder?